airflow/utils/dag_processing.py

```diff
@@ -488,6 +498,25 @@ def _get_log_file_path(self, dag_file_path):
 
         return os.path.join(log_directory, *path_elements)
 
+    def symlink_latest_log_directory(self):
+        """
+        Create symbolic link to the current day's log directory to
+        allow easy access to the latest scheduler log files.
+
+        :return: None
+        """
+        log_directory = self._get_log_directory()
+        latest_log_directory_path = os.path.join(
+            self._child_process_log_directory, "latest")
+        if (os.path.isdir(log_directory)):
+            # if symlink exists but is stale, update it
+            if (os.path.islink(latest_log_directory_path)):
+                if(os.readlink(latest_log_directory_path) != log_directory):
+                    os.unlink(latest_log_directory_path)
+                    os.symlink(log_directory, latest_log_directory_path)
+            else:
+                os.symlink(log_directory, latest_log_directory_path)
```

Comment:
Should probably handle the case where the folder exists but is not a symlink (throw a warning or something and skip creation).